ERCS/erc-7861.md

```diff
@@ -0,0 +1,205 @@
+---
+eip: 7861
+title: ERC-721 Verifiable Credential Extension
+description: An extension to ERC-721 that adds collection-wide verifiable credential properties, this allows native support for onchain credentials.
+author: Valerio Massimo Camaiani (@vmc-crossmint)
+discussions-to: https://ethereum-magicians.org/t/erc-721-verifiable-credential-extension/22562
+status: Draft
+type: Standards Track
+category: ERC
+created: 2025-01-14
+requires: 721
+---
+
+## Abstract
+
+This standard is an extension of [ERC-721](./eip-721.md) that adds verifiable credential properties at the contract level. It allows [ERC-721](./eip-721.md) tokens to natively reference and manage credentials (including W3C Verifiable Credentials or other kinds of off-chain data), by defining mechanisms for storage, encryption, revocation, and verification.
+
+## Motivation
+Many [ERC-721](./eip-721.md) contracts require additional properties to represent verifiable credentials. While it's possible to implement these properties in various ways, it creates an extra effort for third-party platforms to build individualized solutions for each NFT collection.
+
+Having a standard for verifiable credential properties will make it easier for third-party platforms to interact with and validate NFTs that represent verifiable credentials onchain.
+
+Just as NFTs represent digital ownership, verifiable credentials (VCs) represent digital attestations about oneself (identity, education, and more). Blockchains provides an ideal solution for storing VCs and associated claims, offering precise access control, issuer legitimacy, and full lifecycle management for credentials, all while maintaining transparency to safeguard user rights.
+
+This standard addresses key gaps in the W3C framework by defining methods for storage, revocation, retrieval, and presentation, facilitating interoperability within a flexible, standardized structure. It supports custom cryptographic methods for signatures and proofs—from basic issuer signatures to advanced zero-knowledge (ZK) proofs and selective disclosure—allowing adaptability for diverse security requirements.
+
+Additionally, the standard natively supports encryption, crucial for managing sensitive data. It also provides flexibility in credential storage, referencing off-chain data via credentialURI to accommodate any storage solution, from decentralized storage (like IPFS) to centralized systems.
+
+Originally designed with W3C credentials in mind, this standard is broad enough to support any kind of credential, for example content credentials. More generally this framework can be used to link any off-chain, private payload to an NFT.
+
+### Overview
```

Comment:
This overview would be more at home in the specification section, since it deals with the "what" instead of the "why" of this proposal.